core/domain/stats_jobs_one_off.py

```diff
@@ -802,6 +802,19 @@ def entity_classes_to_map_over(cls):
     @classmethod
     def require_non_negative(
             cls, exp_id, exp_version, property_name, value, state_name=None):
+        """Returns the Negative count of the exploration according
```

Comment:
This is wrong. This method does not return anything, and in particular it doesn't return a count.

Reply:
Basically the function is returning the string if the property name is less than zero.

Reply:
This method actually yields the string. So, @seanlip we should include yielded value within `Yields:`.

Reply:
@vibhor98 did you mean to mention me here?

Reply:
@seanlip Yes, I mentioned you here as I wanted to know whether `Yields` will be more suitable here or we should use `Returns` only.

Reply:
Ah I see. Yes, we should use `Yields` instead if the function yields rather than returns stuff.